cucumber/authenticators/features/step_definitions/authn_common_steps.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../../../cucumber/api/features/step_definitions/authz_steps'
```

Comment:
Are you only using a single step definition from this file? If so, I would just duplicate it here and then add a TODO comment and a GitHub issue that say something about finding a good way to share step definitions between different cucumber categories.

Reply:
done